datahub-web-react/src/app/search/EntityGroupSearchResults.tsx

```diff
@@ -23,6 +22,8 @@ interface Props {
     query: string;
 }
 
+const RESULTS_PER_GROUP = 3;
```

Comment:
why 3?

Reply:
This means 3 entities per preview group (before you see "See all X results)... It looks good to me